x-pack/plugins/security_solution/kibana.json

```diff
@@ -22,6 +22,7 @@
   ],
   "optionalPlugins": [
     "encryptedSavedObjects",
+    "lens",
```

Comment:
I'm not sure why some of @patrykkopycinski 's commits are here? All I added in `security_solution` was the `lensIntegration` code